src/lib/ui/device/OnlineStatus.tsx

```diff
@@ -4,20 +4,21 @@ import { OnlineStatusDeviceOfflineIcon } from 'lib/icons/OnlineStatusDeviceOffli
 import { OnlineStatusOnlineIcon } from 'lib/icons/OnlineStatusOnline.js'
 
 export function OnlineStatus(props: { device: LockDevice }): JSX.Element {
```

Comment:
Prefer `export interface OnlineStatusProps` as noted.

Reply:
Updated in https://github.com/seamapi/react/pull/124/commits/72461f87302670be08f6e88bb5968294f9f50b0e